README.md

```diff
@@ -1,16 +1,32 @@
-# elgentos/magento2-prismicio
+# Magento 2 Prismic.io integration
 
-## Magento 2 [Prismic.io](https://prismic.io) integration
-
-We know, we need docs. In the meantime, you can find info over in the wiki; https://github.com/elgentos/magento2-prismicio/wiki
+This package make it possible to create pages within Magento based on the 
+content created in [Prismic.io](https://prismic.io).
 
 ## Installation
 
-```
+This package can easily be installed using composer with the following commands:
+
+```bash
 composer require elgentos/module-prismicio
-php bin/magento s:up
+bin/magento setup:upgrade
 ```
 
-## Configuration
+## Usage
+
+First of all you will need an active Prismic.io account and an API token to
```

Comment:
Perhaps turn "active Prismic.io account" into a link to https://prismic.io/dashboard/signup